src/main/java/eggy/Eggy.java

```diff
@@ -0,0 +1,68 @@
+package eggy;
+
+import eggy.command.Command;
+import eggy.exception.EggyException;
+import eggy.parser.Parser;
+import eggy.storage.Storage;
+import eggy.task.TaskList;
+import eggy.ui.Ui;
+
+/**
+ * The main class of Eggy chatbot.
+ */
+public class Eggy {
+    /** The name of the chatbot. */
+    public final String NAME = "Eggy";
```

Comment:
Should a static modifier be added since `NAME` indicates that it is a constant?

Reply:
I think there might be some confusion between `final` and `static`. `static` doesn't imply constant.

Reply:
Yes, i think `static` can be added here since `NAME` is not associated with instances of the class.